### PR TITLE
Bump version to 1.36.0

### PR DESCRIPTION
**Pull request by bquorning**
I think it’s time for a new release. @rubocop-hq/rubocop-rspec @pirj Do we have any PRs that you with to merge before releasing a new version?

---

Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Added an entry to the [changelog](https://github.com/rubocop-hq/rubocop-rspec/blob/master/CHANGELOG.md) if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Comment by Darhazer** [comment]
Lets include #819 as well

**Comment by Darhazer** [comment]
Also maybe add the changelog entry for #777 and merge as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+## 1.36.0 (2019-09-25)
+
 * Fix `RSpec/DescribedClass`'s error when a `described_class` is part of the namespace. ([@pirj][])
```
Comment (pirj) [inline CHANGELOG.md]: [Bad copy-pasta](https://github.com/rubocop-hq/rubocop-rspec/commit/11659ced7dde8e2ff19efa8debc306de3a0d6e80#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR5), the description is a duplicate of what has been fixed in 1.34.1. Sorry about that.
Should be:
> Fix `RSpec/DescribedClass`'s error when `described_class` is used as part of a constant. ([@pirj][])

Comment (bquorning) [reply CHANGELOG.md]: I’ll fix it.

**Comment by bquorning** [comment]
PR updated.

**Pull request merged**